packages/medusa/src/services/order.ts

```diff
@@ -698,7 +705,7 @@ class OrderService extends TransactionBaseService {
               ),
             ]
           }),
-          cart.shipping_methods.map((method) => {
+          cart.shipping_methods.map(async (method) => {
```

Comment:
todo: the async is not necessary here since a promise is returned

Reply:
this was auto formatted by eslint because of `@typescript-eslint/promise-function-async`

Reply:
yes it is, just wanted to point out that it was returning a promise already just in case. but let's keep it that way 💪